admin/app/views/spree/admin/digital_assets/update.turbo_stream.erb

```diff
@@ -0,0 +1,5 @@
+<%= turbo_render_alerts %>
+<%= turbo_close_modal %>
+
+<%= turbo_stream.update dom_id(@product, :digital_assets),
```

Comment:
@Vegann we don't need to update the entire table, we just need to update the specific record